apps/designer/app/designer/features/style-panel/controls/text/text-control.tsx

```diff
@@ -8,6 +8,8 @@ import type { StyleValue } from "@webstudio-is/css-data";
 import { useState } from "react";
 import { Box, Tooltip } from "@webstudio-is/design-system";
 
+const emptyValue: StyleValue = { type: "string", value: "" };
```

Comment:
Its initially invalid property value. If for example we will add validation not on save but on some other events we will immediately get that its invalid value.

I think we need to pass value undefined here. IT's ok.
And define it inside CSSValueInput as { type: "intermediate", value: "" } as intermediate type can be non valid